packages/audio/espeak-ng/package.mk

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+
```

Comment:
Please add the LE copyright, 2021-present, as technically it's a new (now-ng) package.

Reply:
Please also ensure package files have 644 permissions